pkg/apiserver/admission/admissiontimeout/timeoutadmission_test.go

```diff
@@ -35,13 +34,11 @@ func TestTimeoutAdmission(t *testing.T) {
 	tests := []struct {
 		name string
 
-		timeout         time.Duration
 		admissionPlugin func() (admit admitFunc, stopCh chan struct{})
 		expectedError   string
 	}{
 		{
-			name:    "stops on time",
-			timeout: 50 * time.Millisecond,
+			name: "stops on time",
```

Comment:
because we are passing `context.TODO()`, this test blocks for `13s`, right? If so, I think we should reduce test duration, maybe we can still keep `timeout         time.Duration` as a member variable of `pluginHandlerWithTimeout` so we can manipulate it from a test.

Reply:
Thanks @tkashem, you are right. I changed it to be configurable.